src/vm/jitinterfacegen.cpp

```diff
@@ -224,10 +224,10 @@ void InitJITHelpers1()
         if (GCHeap::UseAllocationContexts())
         {
 #ifdef FEATURE_IMPLICIT_TLS
-            SetJitHelperFunction(CORINFO_HELP_NEWSFAST, JIT_NewS_MP_FastPortable);
-            SetJitHelperFunction(CORINFO_HELP_NEWSFAST_ALIGN8, JIT_NewS_MP_FastPortable);
-            SetJitHelperFunction(CORINFO_HELP_NEWARR_1_VC, JIT_NewArr1VC_MP_FastPortable);
-            SetJitHelperFunction(CORINFO_HELP_NEWARR_1_OBJ, JIT_NewArr1OBJ_MP_FastPortable);
+            SetJitHelperFunction(CORINFO_HELP_NEWSFAST, reinterpret_cast<void*>(JIT_NewS_MP_FastPortable));
```

Comment:
@janvoroli to not prolongate the discussion... I will just add that in the 'extreme use-case' there might be just used an intermediate function, encapsulating the raw one. And all the 

@jkotas actually, it's nice also in COBOL, C and Pascal etc (in general in a language with structural programming).

I will prepare the patch as you ask.